docs/developer-guide.md

```diff
@@ -119,8 +119,47 @@ Here are some key differences between the two, found in [this beautiful post](ht
 - No need to test as many implementation details
 
 
+### Sqitch
+
+We use Sqitch to manage our database with Sqitch via the sqitch.plan in the schema directory.
+To add something to the database:
+- `sqitch add <new database entity> --require <dependency for new entity> --set schema=<schema>` (--require can be repeated for many dependencies)
+To deploy changes:
+- `sqitch deploy <Name of change>` (Name of change is optional, & deploys up to that change. If not set, it will deploy all changes)
+To roll back changes:
+- `sqitch revert <Name of change>` (Name of change is optional, & reverts back to that change. If not set, it will revert all changes)
+
+
+## Incremental Database changes
+
+Releases are tagged in the sqitch plan, ex: `@v1.0.0-rc.7 2020-06-18T18:52:29Z Matthieu Foucault <matthieu@button.is> # release v1.0.0-rc.7`
+Any database entities in the plan before a release are immutable and any changes to them must be done as follows:
+
+### Tables (or other non-idempotent changes)
+
+Changes to tables require creating a new entry in the sqitch plan for that table, for example:
+`sqitch add tables/application_001 --require tables/application`
+This will create a deploy,revert and verify files for tables/application_001
+- In the deploy file, any necessary changes can be made to the table with `alter table`.
+- In the revert file, any changes made in the deploy file must be undone, again with `alter table`
+- The verify file should just verify the existence of the table, and can probably be the same as the original verify file
```

Comment:
This actually follows the recommended approach from sqitch's author https://stackoverflow.com/a/35091927

Reply:
Is my description different? Or did I miss something from the recommended approach? It looks pretty much the same to me.